src/Microsoft.AspNetCore.Blazor/Components/ParameterCollectionExtensions.cs

```diff
@@ -86,9 +86,9 @@ internal static IEnumerable<PropertyInfo> GetBindableProperties(Type targetType)
                         $"name '{propertyName.ToLowerInvariant()}'. Parameter names are case-insensitive and must be unique.");
                 }
 
-                result.Add(propertyName, (ref RenderTreeFrame frame, object target) =>
+                result.Add(propertyName, (object target, object parameterValue) =>
                 {
-                    propertySetter.SetValue(target, frame.AttributeValue);
+                    propertySetter.SetValue(target, parameterValue);
```

Comment:
You might want to consider filling in `PropertyHelper` here if these are cached.

Reply:
Happy to consider it. Since there's extra cost involved in figuring out what exactly that does, and whether it has extra generality that we don't need but has perf implications, and then just in making the code changes, do you happen to know already if there are any functionality differences? Any practical advantages or potential drawbacks?

Reply:
After thinking about this more I think the part of property helper that's valuable is the algorithm for enumerating 'visible' properties.

I don't think the 'set property' logic is useful because I'm not sure it will work well in wasm. PropertyHelper is slightly faster on CoreCLR/NetFx.